src/lib/availability.js

```diff
@@ -0,0 +1,10 @@
+import prisma from "~/lib/prisma";
+
+export const getAvailability = () => {
```

Comment:
Can you rename this function `getAvailabilities()` for consistency with profiles?